pkg/arvo/sys/zuse.hoon

```diff
@@ -2357,6 +2357,144 @@
   ++  blake
     ~%  %blake  ..part  ~
     |%
+    ++  blake3
+      =<
+        =<  hash  :: cuter API
+        =+  [cv=iv flags=0b0]
+        ^?  |%
```

Comment:
This should have

```
~%  %blake3  ..blake3  ~
```

Reply:
hmm, when I try that I get a `-find...blake3`

Reply:
Oh, try putting it inside the `^?`.